Main/tests/MyWalkerTest.kt

```diff
@@ -0,0 +1,12 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+internal class MyWalkerTest {
+    @Test
+    fun `should return string with MyWalker`(){
+        val controlSpeed = 15
+        val actual = MyWalker()
```

Comment:
Actual must be final. But u'r use in the assert actual.walk
It's bad practice
Move calling method to actual initialisation